src/dtw.jl

```diff
@@ -12,6 +12,13 @@ function nanopore_dtw(x::Vector{Float32}, y::Vector{Float32},
     return dtw_bhattacharyya(kmerdists1, kmerdists2)
 end
 
+
+function nanopore_multi_dtw(x::Vector{Vector{Float32}},
+     kmerpath::String = "models/r9.4_70bps.u_to_t_rna.5mer.template.model")
+
```

Comment:
something is missing here, where is changepoint detection? where is kmerdist construction?